src/jmespath.net/Functions/JmesPathFunctionArgument.cs

```diff
@@ -19,7 +19,7 @@ public JmesPathFunctionArgument(JToken token)
 
         public JmesPathFunctionArgument(JmesPathExpression expression)
         {
-            System.Diagnostics.Debug.Assert(expression.IsExpressionType);
+            //System.Diagnostics.Debug.Assert(expression.IsExpressionType);

```

Comment:
Check if this assert should be kept.

Reply:
We can keep this assert for test purpose.